Description: Prevents ant from following symlinks and including library jars
 in debian package
Author: Scott Howard <showard314@gmail.com>
Last-Update: 2017-05-21

Index: arduino/build/build.xml
===================================================================
--- arduino.orig/build/build.xml
+++ arduino/build/build.xml
@@ -92,7 +92,7 @@
   <property name="arduinoOTA-VERSION" value="1.1.1" />
 
   <!-- Libraries required for running arduino -->
-  <fileset dir=".." id="runtime.jars">
+  <fileset dir=".." id="runtime.jars" followsymlinks="no">
     <include name="arduino-core/arduino-core.jar" />
     <include name="app/pde.jar" />
     <include name="app/lib/*.jar" />
